corehq/toggles.py

```diff
@@ -794,7 +794,7 @@ def _commtrackify(domain_name, toggle_is_enabled):
     'Use a SQLite backend for Touchforms',
     TAG_PRODUCT_PATH,
     [NAMESPACE_DOMAIN],
-    0.1,
+    0.5,
```

Comment:
this means this will be turned on for 50% of all domains, right? am curious if we've run the sms/ivr test suites with this backend (it has to be run manually due to touchforms dependency)?